tests/testthat/test_plotSpatialContext.R

```diff
@@ -0,0 +1,160 @@
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_color_by = "name")
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_color_by = "n_group")
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_color_by = "n_cells")
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_size_by = "n_cells")
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_size_by = "n_group")
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_label_repel = FALSE)
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_label_color_by = "name")
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_label_color_by = "n_group")
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  p <- plotSpatialContext(sce, group_by = "ImageNb", node_label_color_by = "n_cells")
+  expect_s3_class(p, "ggraph")
+  expect_silent(print(p))
+  expect_equal(p$data$name, sort(unique(sce$spatial_context)))
+  expect_equal(p$data$length, sapply(str_split(sort(unique(sce$spatial_context)),"_"),length))
+  
+  #return data - tests
```

Comment:
Could you add one test here that checks the actual entries in the output? Just to make sure that the entries are always the same.